test/jdk/javax/net/ssl/DTLS/InvalidRecords.java

```diff
@@ -36,27 +36,32 @@
 
 import java.net.DatagramPacket;
 import java.net.SocketAddress;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
  * Test that if handshake messages are crasged, the handshake would fail
```

Comment:
crasged?  Was that supposed to be "changed?"

Reply:
I fixed the typo.